lib/large_object_store.rb

```diff
@@ -19,7 +20,20 @@ def initialize(store)
 
     def write(key, value, options = {})
       value = Marshal.dump(value)
-      value = Zlib::Deflate.deflate(value) if options.delete(:compress)
+
+      compress_limit = options[:compress_limit] || DEFAULT_COMPRESS_LIMIT
```

Comment:
For a Rails wrapper, consistency with Rails options behavior isn't a bad thing...